info.go

```diff
@@ -182,3 +182,19 @@ func scanFdInfoReader(r io.Reader, fields map[string]interface{}) error {
 
 	return nil
 }
+
+// EnableStats starts the measuring of the runtime (in ns)
+// and run counts.
+//
+// Requires at least 5.8.
+func EnableStats() error {
+	attr := internal.BPFEnableStatsAttr{
+		StatsTyp: internal.BPF_STATS_RUN_TIME,
+	}
+
+	return internal.BPFEnableStats(&attr)
+}
+
+var canEnableStats = internal.FeatureTest("can enable stats", "5.8", func() error {
```

Comment:
Sorry. I didn't realize that there was no way to disable runtime stats after they were enabled. I guess this feature test is actually kind of useless. I recommend removing it.

Reply:
Agree. Is it possible to return a wrapped ErrNotSupported based on the return value from BPFEnableStats?

For future reference, you need to return ErrNotSupported from feature tests in some form, otherwise there isn't much point to it.

Reply:
i have removed the feature test again.